### PR TITLE
Add missing fields

### PR DESCRIPTION
**Pull request by truechernyshov**
Add missing properties to Subscription, Invoice.

### CODE REVIEW

**Review by subenson — DISMISSED** [review]

**Review by justnero — DISMISSED** [review]

File: src/Entities/PaymentInstruments/PaymentCardPaymentInstrument.php [hunk 1]
```diff
@@ -84,4 +84,20 @@ protected function methodName()
...
+        return $this->getAttribute('expirationReminderTime');
+    }
+
+    /**
+     * @return int
+     */
+    public function getExpirationReminderNumber()
+    {
+        return $this->getAttribute('expirationReminderNumber');
+    }
```
Comment (ashkarpetin) [inline src/Entities/PaymentInstruments/PaymentCardPaymentInstrument.php:102]: I don't think that's correct adding it here.

**Review by ashkarpetin — CHANGES_REQUESTED** [review]
Methods from PaymentCardPaymentInstrument should be moved to `CommonPaymentInstrument` class

File: src/Entities/PaymentInstruments/PaymentCardPaymentInstrument.php (hunk 1 above)
Comment (truechernyshov) [reply src/Entities/PaymentInstruments/PaymentCardPaymentInstrument.php]: Removed it.

**Review by ashkarpetin — APPROVED** [review]

**Review by 2e3s — APPROVED** [review]

**Comment by ashkarpetin** [comment]
Thanks!

**Pull request merged**